app/(dashboard)/appSidebar.tsx

```
@@ -32,6 +32,7 @@ import {
   useSidebar,
 } from "@/components/ui/sidebar";
 import { api } from "@/trpc/react";
+import { Skeleton } from "@/components/ui/skeleton";
```

ESLint

[error] Replace `api·}·from·"@/trpc/react";⏎import·{·Skeleton·}·from·"@/components/ui/skeleton` with `Skeleton·}·from·"@/components/ui/skeleton";⏎import·{·api·}·from·"@/trpc/react`

(prettier/prettier)